test/e2e/README.md

```diff
@@ -46,13 +46,31 @@ OR follow this doc for deploying go https://golang.org/doc/install
 
 ### Trigger through CLI
 * Launching CI tests is as simple as below.
-```console
+```shell
 $ kubectl config use-context <<cluster-under-test-context>>
 $ ginkgo -r -v ci -timeout=2h -- -yk-namespace "yunikorn" -kube-config "$HOME/.kube/config"
 ```
 
-* Launching all the tests can be done as..
-```console
+* Launching all the tests can be done as.
+```shell
 $ ginkgo -r -v -timeout=2h -- -yk-namespace "yunikorn" -kube-config "$HOME/.kube/config"
 
 ```
+
+* Launching all the tests in specified e2e folder.
+e.g. test/e2e/user_group_limit/
+```shell 
+$ cd test/e2e/
+$ ginkgo -r user_group_limit -v -- -yk-namespace "yunikorn" -kube-config "$HOME/.kube/config"
+```
+
+* Launching specified test.
+e.g. The first augument of ginkgo.It()
+
+ginkgo.It( "Verify_maxapplications_with_a_specific_group_limit" , ... )
```

Comment:
<img width="1096" alt="image" src="https://github.com/apache/yunikorn-k8shim/assets/26764036/89313d5f-31a9-4139-8b34-a457c99eb317">

The format seems broken, would it be better to just use:
* Launching specified test. e.g. Run test with spec name "Verify_maxapplications_with_a_specific_group_limit"

Reply:
Thanks @chenyulin0719  for reviewing. fair point 👍